Tests/CommandsTests/TestToolTests.swift

```diff
@@ -313,4 +313,17 @@ final class TestToolTests: CommandsTestCase {
             }
         }
     }
+
+    // TODO: This test should be moved into `ResourceTests.swift` in the
+    // `FunctionalTests` scheme when the `FunctionalTests` scheme is re-enabled.
+    func testResourceBundleInClangPackageWhenRunningSwiftTest() throws {
```

Comment:
I think we can move it as part of this PR, otherwise it can cause the same type of issue that brought us to disable `FunctionalTests`.